app/grandchallenge/workstation_configs/templates/workstation_configs/workstationconfig_detail.html

```diff
@@ -0,0 +1,84 @@
+{% extends "base.html" %}
+{% load static %}
+{% load url from grandchallenge_tags %}
+{% load user_profile_link from profiles %}
+{% load workstations %}
+{% load guardian_tags %}
+
+{% block breadcrumbs %}
+    <ol class="breadcrumb">
+        <li class="breadcrumb-item"><a href="{% url 'home' %}">Home</a></li>
+        <li class="breadcrumb-item"><a href="{% url 'workstation-configs:list' %}">Workstation Configs</a></li>
+        <li class="breadcrumb-item active" aria-current="page">{{ object.title }}
+        </li>
+    </ol>
+{% endblock %}
+
+{% block content %}
+    {% get_obj_perms request.user for object as "config_perms" %}
+
+    <h1>Workstation Config {{ object.title }}</h1>
+
+    {% if object.description %}
+        <p>{{ object.description }}</p>
+    {% endif %}
+
+    <div class="my-3">
+        <div class="d-flex justify-content-between">
+            <div class="font-weight-bold">Window presets</div>
+            <div>
+                {% for x in object.window_presets.all %}
+                    <div>{{ preset }}</div>
+                {% endfor %}
+            </div>
+        </div>
+        <div class="d-flex justify-content-between">
```

Comment:
The separation between key and value makes this hard to read:

![Screenshot_20191210_145428](https://user-images.githubusercontent.com/12661555/70535285-12d3cb80-1b5d-11ea-94ea-604eaa8f1bd8.png)